xml_inscriptions/metadata_only/CA.Malibu.JPGM.ETR.82.AD.25.xml

```diff
@@ -0,0 +1,164 @@
+                        <!-- arc, grc, he, lat, und (unknown),  zxx (non-linguistic content) -->
+                        <msItem class="#honorific">
+                            <!-- short summary of the content of an inscription -->
+                            <p>Inscription on a bulla.</p>
+                        </msItem>
+                    </msContents>
+
+                    <!--  *************************************  <physDesc>  ********************************* -->
+                    <physDesc>
+
+                        <!--  *************************************  <objectDesc>  *********************************  -->
+
+                        <objectDesc ana="#instrumentum">
+                            <supportDesc ana="#clay">
+                                <support>
+                                    <p> Terracotta bulla or seal</p>
+                                    <dimensions type="surface" unit="cm">
+                                        <height>5.8</height>
+                                        <width>4.9</width>
+                                        <depth>2.2</depth>
+                                    </dimensions>
+                                </support>
+                                <condition ana="#complete">
+                                    <p/>
+                                </condition>
+                            </supportDesc>
+                            <!--  *************************************  <layoutDesc>  ********************************* -->
+                            <layoutDesc>
+                                <layout columns="0" writtenLines="0">
+                                    <p/>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+
+                        <!--  *************************************  <handDesc>  ********************************* -->
+
+                        <handDesc hands="1">
+                            <handNote ana="#impressed"> </handNote>
+                        </handDesc>
+
+                        <!--  *************************************  <decoDesc>  *********************************     -->
+                        <decoDesc>
+                            <decoNote>
+                                <p/>
+                            </decoNote>
+                        </decoDesc>
+                    </physDesc>
+
+                    <!--  *************************************  <history>  *********************************  -->
+
+                    <history>
+
+                        <summary>
+                            <rs/>  <!-- content can be "fake" or blank -->
+                        </summary>
+
+                        <origin>
+                            <date notBefore="0501" notAfter="0599">6th century CE</date>
+                            <placeName ref="asia.asia_minor">Persia</placeName>
```

Comment:
Persia isn't technically in Asia Minor anymore, right? Shouldn't it just be asia.persia?